curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-provide-explanations-when-helping-others/6626273eb00c031f4988f129.md

```diff
@@ -7,7 +7,7 @@ dashedName: task-3
 
 # --description--
 
-Frustration is a feeling of being upset or annoyed because of inability to change or achieve something. For example, it can be frustrating when you're trying to learn a new skill but find it difficult. This emotion often comes when something is hard or doesn't go as planned.
+Frustration is the feeling of being upset or annoyed because of inability to change or achieve something. For example, it can be frustrating when you're trying to learn a new skill but find it difficult. This emotion often comes when something is hard or doesn't go as planned.
```

Comment:
```suggestion
`Frustration` is the feeling of being upset or annoyed because of inability to change or achieve something. For example, it can be `frustrating` when you're trying to learn a new skill but find it difficult. This emotion often comes when something is hard or doesn't go as planned.
```
Not sure about the second backticks. What do you think?